app/services/import_actuals.rb

```diff
@@ -129,6 +137,15 @@ def initialize(row)
       @activity = @attributes.delete(:activity)
     end
 
+    # We must have precisely one value, and we will check if
+    # the other is empty when we validate the value it contains
+    def actual_class
+      return Actual if @row["Refund Value"].nil?
+      return Refund if @row["Actual Value"].nil? || @row["Actual Value"] == "0.00"
+      @errors[:value] = [@row["Actual Value"], I18n.t("importer.errors.actual.non_numeric_value")]
+      @errors[:refund_value] = [@row["Refund Value"], I18n.t("importer.errors.actual.non_numeric_value")]
```

Comment:
Busy wondering if we'll get away with one of these, perhaps with a virtual attribute name like `@errors[:refund_or_actual_value]`?

Reply:
Might need a little help understanding the virtual attribute name, but sure!

Reply:
I mean: the attribute doesn't "exist" as such, it's just a placeholder meaning "these two fields". These aren't real ActiveModel errors, they're just our way of communicating a thing. At present we communicate "two of these things are wrong" for our upload errors UI (so two errors for the given row) where we might be better served by communicating "you only needed to supply one of these" as a single "error" applying to a "field" `:refund_or_actual_value` (which doesn't exist but would appear in the errors list as "Refund or actual value").

I don't honestly know if I've made it worse with this "explanation" 😆

Reply:
No, that's clear, and sounds good.